datapreprocess/ray/tokenize_shuffle.py

```diff
@@ -375,7 +406,7 @@ def human_to_bytes(s):
         ray.init(args.ray_address, runtime_env=runtime_env)
     num_nodes = len(ray.nodes())
     # TODO  support multiple inputs
-    input_paths = glob_files(args.input, suffix=".jsonl")
+    input_paths = glob_files(args.input, suffix=f".{args.dataset_type}")
```

Comment:
I think it would be good to split dataset_type and suffix - @jeffreywpli has told me that it is convenient to glob e.g. particular jsonls based on suffix.

Reply:
I see... maybe there's a way to *just* use "suffix"? Can we make the point that a tar dataset is one with `"tar" in args.suffix` or is this messy?

Reply:
That might be better - maybe something like `dataset_type = args.suffix.split(".")[-1]`?
I think it's reasonable to assume that the extension is correct.